charts/kube-prometheus-stack/templates/prometheus-operator/clusterrole.yaml

```diff
@@ -78,4 +78,12 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
```

Comment:
Can we put also an `if` clause around this and have this deployed only when the K8s version is `>=1.22`?

Reply:
According to the [k8s docs](https://kubernetes.io/docs/concepts/services-networking/endpoint-slices/) endpoint slices are a 1.21 feature

Reply:
But the chart claims to be k8s 1.16 compatbile: https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack#prerequisites

Reply:
Case added, please check